packages/web-components/fast-foundation/src/picker/picker-list-item.ts

```diff
@@ -0,0 +1,95 @@
+import { attr, html, HTMLView, observable, ViewTemplate } from "@microsoft/fast-element";
+import { keyEnter } from "@microsoft/fast-web-utilities";
+import { FoundationElement } from "../foundation-element";
+
+const defaultContentsTemplate: ViewTemplate<PickerListItem> = html`
+    <template>
+        ${x => x.value}
+    </template>
+`;
+
+/**
+ * A picker list item Custom HTML Element.
+ *
+ * @alpha
+ */
+export class PickerListItem extends FoundationElement {
+    /**
+     * The underlying string value of the item
+     *
+     * @alpha
+     * @remarks
+     * HTML Attribute: value
+     */
+    @attr({ attribute: "value" })
+    public value: string;
+
+    /**
+     *  The template used to render the contents of the list item
+     *
+     * @alpha
+     */
+    @observable
+    public contentsTemplate: ViewTemplate;
+    private contentsTemplateChanged(): void {
+        if (this.$fastController.isConnected) {
+            this.updateView();
+        }
+    }
+
+    private customView: HTMLView | undefined;
+
+    /**
+     * @internal
+     */
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.updateView();
+    }
+
+    /**
+     * @internal
+     */
+    public disconnectedCallback(): void {
+        this.disconnectView();
+        super.disconnectedCallback();
+    }
+
+    public handleItemKeyDown = (e: KeyboardEvent): boolean => {
```

Comment:
Since the handlers are called as properties on `PickerListItem` objects, the names of the methods probably don't need `Item` in their names.